src/box/sql.c

```diff
@@ -1415,16 +1415,15 @@ vdbe_field_ref_create(struct vdbe_field_ref *ref, uint32_t capacity)
 	ref->field_capacity = capacity;
 }
 
-ssize_t
+size_t
 sql_index_tuple_size(struct space *space, struct index *idx)
 {
 	assert(space != NULL);
 	assert(idx != NULL);
 	assert(idx->def->space_id == space->def->id);
-	ssize_t tuple_count = index_size(idx);
-	ssize_t space_size = space_bsize(space);
-	ssize_t avg_tuple_size = tuple_count != 0 ?
-				 (space_size / tuple_count) : 0;
+	size_t tuple_count = index_size(idx);
+	size_t space_size = space_bsize(space);
+	size_t avg_tuple_size = tuple_count != 0 ? space_size / tuple_count : 0;
```

Comment:
This would probably result in another SVACE error because `index_size` and `space_bsize` return a value of type `ssize_t`. What are you trying to accomplish here? What possible problems would this patch solve apart from suppressing SVACE? If it can't lead to a bug, then just mark the issue as "Won't fix" or "False positive".

Reply:
I thought that both return size_t values, but I was wrong - `index_size()` returns a value of type 'ssize_t`. However, this is not the case for `space_bsize()`. But I agree, I will mark this issue as "Won't fix".